handler/openid/flow_explicit_token_test.go

```diff
@@ -113,6 +113,20 @@ func TestExplicit_PopulateTokenEndpointResponse(t *testing.T) {
 				store.EXPECT().GetOpenIDConnectSession(nil, gomock.Any(), areq).AnyTimes().Return(r, nil)
```

Comment:
We should check here if the hash has been set correctly

Reply:
Thanks, I will apply it.